functions/venue.js

```diff
@@ -86,7 +87,11 @@ const IFRAME_TEMPLATES = [
 const ZOOM_URL_TEMPLATES = [VenueTemplate.artcar, VenueTemplate.zoomroom];
 
 // @debt unify this with HAS_REACTIONS_TEMPLATES in src/settings.ts + share the same code between frontend/backend
```

Comment:
[NOTE]
Yeah, maybe these debt comments should become "move to templateSettings.ts or similar"

Reply:
do we have some sort of shared settings for `functions` and `src` atm?

Reply:
Nothing. You can check https://doesitsparkle.slack.com/archives/C028JG5TUMC/p1635776432013900?thread_ts=1635768348.012700&cid=C028JG5TUMC for similar/same discussion